charts/minecraft/templates/extraports-ing.yaml

```diff
@@ -9,8 +9,7 @@ kind: Ingress
 metadata:
   name: {{ $serviceName }}
   {{- if .ingress.annotations }}
-  annotations:
-    {{- toYaml .ingress.annotations | nindent 4 }}
+  annotations: {{- toYaml .ingress.annotations | nindent 4 }}
```

Comment:
Hmm, does `toYaml` know to render that with inline object syntax?

Reply:
that's what `nindent` does, the `n` is for newline, then indent.

Reply:
Sorry, I missed that. You can ignore my other comment since that'll introduce way too much change.